raiden/transfer/architecture.py

```diff
@@ -255,7 +259,9 @@ def dispatch(self, state_changes: List[StateChange]) -> Tuple[ST, List[List[Even
 
         # The state objects must be treated as immutable, so make a copy of the
         # current state and pass the copy to the state machine to be modified.
+        before_copy = time.time()
         next_state = deepcopy(self.current_state)
+        log.debug("Copy state before applying state changes", duration=time.time() - before_copy)
```

Comment:
nit: what about *Copying state* or *Copied state*?

Reply:
Yes, `copied` is actually more precise. I'll change it.